### PR TITLE
[DA-2218] Dont split phrases

### PR DESCRIPTION
**Pull request by dopc**
* The module accepts `list of list of tokens`
* `CountVectorizer` is modified in BERTopic

### CODE REVIEW

File: bertopic/_bertopic.py [hunk 1]
```diff
@@ -160,6 +160,10 @@ def __init__(self,
         self.nr_topics = nr_topics
         self.min_topic_size = min_topic_size
         self.calculate_probabilities = True
+        if type(cluster_selection_epsilon) is int:
+            cluster_selection_epsilon = float(cluster_selection_epsilon)
+        if type(cluster_selection_epsilon) is not float or cluster_selection_epsilon < 0.0:
+            raise ValueError('Epsilon must be a float value greater than or equal to 0!')
```
Comment (zafercavdar) [inline bertopic/_bertopic.py]: No need to have this check. HDBSCAN should be already checking these 2 conditions.

File: bertopic/_bertopic.py [hunk 2]
```diff
@@ -797,7 +803,10 @@ def _extract_topics(self, documents: pd.DataFrame):
         Returns:
             c_tf_idf: The resulting matrix giving a value (importance score) for each word per topic
         """
-        documents_per_topic = documents.groupby(['Topic'], as_index=False).agg({'Document': ' '.join})
+        # documents_per_topic = documents.groupby(['Topic'], as_index=False).agg({'Document': ' '.join})
+        documents_per_topic = documents.groupby(['Topic'], as_index=False).agg(
+            {'Document': lambda topic: [token for doc in topic for token in doc]})
```
Comment (zafercavdar) [inline bertopic/_bertopic.py]: When grouped by `Topic`, does it produce `list of list of list of tokens`?

**Review by zafercavdar — CHANGES_REQUESTED** [review]

File: bertopic/_bertopic.py (hunk 1 above)
Comment (dopc) [reply bertopic/_bertopic.py]: Done!

File: bertopic/_bertopic.py (hunk 2 above)
Comment (dopc) [reply bertopic/_bertopic.py]: Done!

**Pull request merged**